src/xpcc/architecture/platform/driver/core/cortex/driver.xml

```diff
@@ -16,11 +17,23 @@
 		<parameter name="hardfault_handler_led_port" type="enum" values="A;B;C;D;E;F;G;H;I;J;K">A</parameter>
 		<parameter name="hardfault_handler_led_pin" type="int" min="0" max="15">0</parameter>
 
-
 		<parameter name="free_rtos_support" type="bool">false</parameter>
 		<parameter name="free_rtos_frequency" type="int" min="1000" max="10000">1000</parameter>
 
 
+		<!-- lpc -->
+		<template device-platform="lpc" out="linkerscript.ld">stm32/stm32_ram.ld.in</template>
+		<!-- stm32f0, stm32f1, stm32f2 -->
+		<template device-family="f0|f1|f2" out="linkerscript.ld">stm32/stm32_ram.ld.in</template>
+		<!-- stm32f3 -->
+		<template device-family="f3" device-name="303|328|334|358|398" out="linkerscript.ld">stm32/stm32_iccm.ld.in</template>
+		<template device-family="f3" device-name="301|302|318|378|373" out="linkerscript.ld">stm32/stm32_ram.ld.in</template>
+		<!-- stm32f4 -->
+		<template device-family="f4" device-name="405|407|415|417|427|429|437|439|469|479" out="linkerscript.ld">stm32/stm32_dccm.ld.in</template>
+		<template device-family="f4" device-name="401|410|411|446" out="linkerscript.ld">stm32/stm32_ram.ld.in</template>
```

Comment:
@dergraaf At this stage I really wanted to be able to look at the content of the device file to check if it contains the `ccm` memory section. I came to the conclusion that this file should be replaced by a python class with builds this together dynamically. Then I realized we really need a module system :-(